packages/core/src/kernelCoreTemplate.js

```diff
@@ -104,12 +104,23 @@
         .forEach(proto =>
           props.push(...Object.getOwnPropertyNames(proto)))
 
+      for (let i = 0; i < extraPropsToAvoid.length; i++) {
+        const prop = extraPropsToAvoid[i]
```

Comment:
Just to make sure: it seems we can trust the input here as coming from the trusted initialization code, right?